contribute/labels-of-issues.md

```diff
@@ -0,0 +1,115 @@
+# List of labels for issues and PRs on OpenEBS Project
+
+This document provides a list of labels and their purpose. These labels can be attached to issues and PRs. The aim behind having such list of labels is, we can sort and distinguish issues. This list will help new community members to dive-in OpenEBS if they wish to contribute to project or investigate the problem.
+
+Use case:
+> I am a beginner, I want to contribute to OpenEBS project and I am not sure from where to begin?
+
+Answer:
+> Start looking into the issues tagged with labels such as `help/small` , `size/XS`, `kind/unit-test` etc. Solve such issues and raise PR.
+
+---
+
+## Kind
+
+Labels prefixed with *kind* indicate the type of issue and PR. Suppose, you want to report a bug then you can tag your issue with `kind/bug` label.
+
+| Label | Description |
+|---|---|
+| kind/api  | Issue/PR related to API  |
+| kind/bug |  If there’s an error, failure in workflow, fault etc.|
+| kind/feature  | New / existing / required feature or enhancement |
+| kind/improve | Needs improvement |
+| kind/e2e-test | Need/Add end to end test |
+| kind/unit-test | Need/Add Unit test|
+| kind/security  | Security-related concerns  |
+| kind/testing  |  Needs/Adds tests |
+| kind/good-first  |  beginner friendly |
+
+---
+
+## Help
+
+Labels prefixed with *help* implies that issue needs some help from community/users. Further, you can adjust the level of help required, by specifying small, medium, large etc.
+
+| Label | Description |
+|---|---|
+| help/small  | Need small help on docs (beginner level)|
+| help/medium | Need help to fix the code ( Intermediate level) |
+| help/large | Need help on design changes and code review (Expert level) |
+
+---
+
+## Area
+
+Labels prefixed with *area* indicates that which area issue or PR belong. Here, area implies component of the OpenEBS project. It will be easier to sort an issue based on the area of user's interest.
+
+| Label | Description |
+|---|---|
+| area/maya  | Maya Specific  |
+| area/k8s | Kubernetes specific   |
+| area/ci  | Continuous integration specific  |
+| area/high-availability  | High availability  |
+| area/storage | Storage specific  |
+| area/machine-learning  |  Machine learning specific |
+
+---
+
+## Size
+
+Labels prefixed with *size* defines the size of change done in particular PR.
```

Comment:
sounds like this is saying it's the size on a PR
but the label should be added on a issue, which estimates what size a future PR would eventually be?

Reply:
I think it's that, once decided for pickup, someone can add the size on the issue?

Reply:
Then slightly reworking as to me it's reading the size already done rather than an estimate

```suggestion
Labels prefixed with *size* defines the expected size of the changes required to fix the issue.
```